Sage/carbon
# chore(mount-in-app): remove classic theme

removal of classic theme from the mount in app component.

BREAKING CHANGE: Removal of classic theme from the mount in app component. Component no longer
supports classic theme.

fixes FE-3116

### Proposed behaviour
- Component should not support classic theme
- Documentation is now included in an MDX example
- Stories file is now in the CSF

### Current behaviour
- Component supports classic theme
- Documentation complies with our previous standards
- Stories file is in the depreciated storiesOf format

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout FE-3116-mount_in_app_remove_classic_theme`
- `npm start` to start Storybook
- Navigate to the Mount In App story
- No classic theme example should be present
- Documentation should be in MDX format
